Jax/Page/ModControls.php

```diff
@@ -109,7 +108,7 @@ private function load(): void
 
         if (!$this->request->isJSAccess()) {
             header('Content-Type: application/javascript; charset=utf-8');
-            header('Expires: ' . gmdate('D, d M Y H:i:s', time() + 2_592_000) . ' GMT');
+            header('Expires: ' . gmdate('D, d M Y H:i:s', Carbon::now()->getTimestamp() + 2_592_000) . ' GMT');
```

Comment:
probably can use carbon to add the what is that, a month? and make this readable instead of a magic number